tests/fixtures/lowering/base_tuple_lowered.fe

```diff
@@ -1,52 +1,52 @@
-struct tuple_u256_bool:
+struct __Tuple0:
```

Comment:
Did you consider alterations to the current naming scheme that would preserve the readability better? E.g. we could encode the opening and closing parenthesis with some character like an extra `_`.


```
((A, B), C, D) // _tuple__tuple_A_B__C_D_
((A, B, C), D) // _tuple__tuple_A_B_C__D_
```

Reply:
Yeah, I was thinking that that's *less* readable in the case of a couple nestings, but it's better in the simpler/common cases, so I'll switch it back.

Reply:
yes, I guess having to differentiate between two easy to confuse tuples will be rather rare so maybe it's better to keep it readable for the common cases.

Reply:
It's now 
```
(A, B) => tuple_A_B_
(A, (B, C), D) => tuple_A_tuple_B_C__D_
``` 
ie, replace parens and commas with underscores, and prefix with "tuple"

Reply:
Looks good :+1: